### DISCUSSION TITLE
Incohérence entre exemple du read.me et le swagger

### DISCUSSION THREAD

**Discussion by AdamBoudjemaa** [before-interop/anomalieAdresse#45] [General]
Bonjour, 

Sur le read.me on a cet exemple :
![Capture d’écran du 2023-10-19 12-14-58](https://github.com/before-interop/anomalieAdresse/assets/148445439/5700e198-5881-43ce-b947-af3b9430eaec)

On a un tableau d'objet relatedEntity et aussi un champ building.address.city qui contient une "**description**"

Alors que sur le swagger, dans le request body, on ne retrouve pas l'objet relatedEntity et pour le champ building.address.city , il ya uniquement  une "**value**"

![Capture d’écran du 2023-10-19 12-21-53](https://github.com/before-interop/anomalieAdresse/assets/148445439/39103cdd-873e-404a-b9a7-6964fdad5548)

Sur quelle base devront nous travailler ?

Merci, 
Adam

**Comment by ggrebert** [comment]
L'exemple du README n'est pas à jour.
Faut que je trouve le temps pour le remplacer.
